h2o-jetty-9/src/main/java/water/webserver/jetty9/Jetty9Helper.java

```diff
@@ -73,6 +72,7 @@ Server createJettyServer(String ip, int port) {
         final ArrayList<ConnectionFactory> connectionFactories = new ArrayList<>();
         ConnectionFactory connectionFactory = new HttpConnectionFactory();
         final HttpConfiguration httpConfiguration = new HttpConfiguration();
+        httpConfiguration.setSendServerVersion(true);
```

Comment:
I think this is good idea - and we can add the same to jetty8 version in this PR, WDYT?

Reply:
We already have this in Jetty 8. This is only done in Jetty 9 in a completely different way, like this ^^^.